.github/workflows/build.yml

```diff
@@ -42,7 +42,14 @@ jobs:
       # Build
       - name: Build
         run: bash ./gradlew build --stacktrace
-      
+
+      #uiTests
+      - name: run tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29
```

Comment:
is this api level correct?
I don't quite understand the stack trace, but I see a "unknown API level" skip warning
![image](https://user-images.githubusercontent.com/11148726/103237874-e7c34200-4940-11eb-9502-01ebbe31a3f5.png)

Reply:
I am not a 100% sure but I think this means that the app will not run on running API level 30 and above (i.e. android 10+). Is this expected or should I raise another issue?